MekHQ/data/scenariotemplates/LegacyAtB/ScenarioManifest.xml

```diff
@@ -0,0 +1,92 @@
+<scenarioManifest>
+	<scenarioFileNames>
+	<entry>
+		<key>10</key>
+		<value>AceDuel.xml</value>
+	</entry>
+	<entry>
+		<key>13</key>
+		<value>AlliedTraitors.xml</value>
+	</entry>
+	<entry>
+		<key>17</key>
+		<value>AllyRescue.xml</value>
+	</entry>
+	<entry>
+		<key>11</key>
+		<value>Ambush.xml</value>
+	</entry>
+	<entry>
+		<key>0</key>
+		<value>BaseAttack.xml</value>
+	</entry>
+	<entry>
+		<key>4</key>
+		<value>Breakthrough.xml</value>
+	</entry>
+	<entry>
+		<key>2</key>
+		<value>Chase.xml</value>
+	</entry>
+	<entry>
+		<key>12</key>
+		<value>CivilianHelp.xml</value>
+	</entry>
+	<entry>
+		<key>18</key>
+		<value>CivilianRiot.xml</value>
+	</entry>
+	<entry>
+		<key>20</key>
+		<value>ConvoyAttack.xml</value>
+	</entry>
+	<entry>
+		<key>19</key>
+		<value>ConvoyRescue.xml</value>
+	</entry>
+	<entry>
+		<key>1</key>
+		<value>Extraction.xml</value>
+	</entry>
+	<entry>
+		<key>5</key>
+		<value>HkeyeAndSeek.xml</value>
+	</entry>
+	<entry>
+		<key>3</key>
+		<value>HoldTheLine.xml</value>
+	</entry>
+	<entry>
+		<key>9</key>
+		<value>OfficerDuel.xml</value>
+	</entry>
+	<entry>
+		<key>21</key>
+		<value>PirateFFA.xml</value>
+	</entry>
+	<entry>
+		<key>14</key>
+		<value>PrisonBreak.xml</value>
+	</entry>
+	<entry>
+		<key>8</key>
+		<value>Probe.xml</value>
+	</entry>
+	<entry>
+		<key>7</key>
+		<value>ReconRakey.xml</value>
```

Comment:
ReconRakey.xml?

Reply:
These scenario definitions wound up being unused - I was originally planning to just convert the legacy AtB scenarios to the template mechanism, but then the project got a little out of control.